Druid-Tests/DruidTestRTLCompiler.class.st

```diff
@@ -2261,9 +2240,9 @@ DruidTestRTLCompiler >> gen_primitiveIntegerDivide [
 	self
 		DivR: ClassReg
 		R: TempReg
-		Quo: ClassReg
-		Rem: TempReg.
-	self MoveR: ClassReg R: ReceiverResultReg.
+		Quo: TempReg
+		Rem: ClassReg.
+	self MoveR: TempReg R: ReceiverResultReg.
```

Comment:
We should check why the allocator is not deterministic... We are probably using a Set somewhere